### PR TITLE
refactor(movies): async/await for controller tests

### PR DESCRIPTION
**Pull request by robintsunny**
What: a/a for controller tests
why: replacing promises with a/a

### CODE REVIEW

File: test/plugins/features/movies/controller.test.js [hunk 1]
```diff
@@ -6,13 +6,12 @@ describe('movie controller', () => {
...
+    it('creates a movie', async () => {
       const payload = { title: 'WALL-E' };
 
-      return Controller.create(payload)
-      .then((movie) => {
-        expect(movie.get('title')).to.eql(payload.title);
-      });
+      const movie = await Controller.create(payload)
+      expect(movie.get('title')).to.eql(payload.title);
+
```
Comment (sjl2) [inline test/plugins/features/movies/controller.test.js:15]: nit: extra new-line.

**Review by sjl2 — APPROVED** [review]
LGTM

File: test/plugins/features/movies/controller.test.js (hunk 1 above)
Comment (robintsunny) [reply test/plugins/features/movies/controller.test.js:15]: fixed

**Review by jesscxu — APPROVED** [review]
LGTM

**Pull request merged**